app/views/submissions/_form.html.erb

```diff
@@ -45,14 +45,18 @@
 			  									 id: 'upload-userManuscript-hidden' %>
 			  <%= f.file_field :manuscript,
 			  								 class: 'form-control-file',
-			  								 id: 'upload-userManuscript' %>
+											   id: 'upload-userManuscript',
+											   accept: 'application/pdf' %>
 			  <%= f.hidden_field :detach_user_manuscript,
 			  									 value: false %>
+			  <button id="userManuscript-uploadButton-btn" class="btn btn-sm btn-outline-secondary"><%= t('basics.file') %></button>
```

Comment:
Please keep within 80 characters and use intendation. There are some lines between 70 and 87 where the same applies.